docs/cmd/kn_source_apiserver_create.md

```diff
@@ -0,0 +1,43 @@
+## kn source apiserver create
+
+Create a Kubernetes API Server event source
+
+### Synopsis
+
+Create a Kubernetes API Server event source
+
```

Comment:
Here you could explain a bit the different modes and what a controller is, which kinds of events are delivered (== all), etc.

Reply:
done